### PR TITLE
[#38] Create UserUiModel with fake data

### PR DESCRIPTION
**Pull request by doannimble**
https://github.com/nimblehq/flutter_templates/issues/38

## What happened 👀

- Create Repository UI Model
- Create fake data
- Add List Repository at Home screen

## Insight 📝

Run and view Home Screen

### CODE REVIEW

File: lib/model/ui/user_ui_model.dart [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+  UserUiModel({
+    required this.avatarUrl,
+    required this.name,
+    required this.description,
+  });
+
+  static List<UserUiModel> fakeData() {
+    return [
+      UserUiModel(
```
Comment (markgravity) [inline lib/model/ui/user_ui_model.dart:14]: What do you think about using a loop here?
It's a good example when creating fake data, cause I was faced with an issue in the varun project that we have to keep repeat updates when we add/remove/change a field in an `UiModel` 😢

File: lib/main.dart [hunk 2]
```diff
@@ -44,26 +45,53 @@ class MyHomePage extends StatelessWidget {
...
-        ),
+          ),
+          SizedBox(height: 24),
+          Expanded(child: _buildRepositories())
+        ],
       ),
     );
   }
+
+  Widget _buildRepositories() {
```
Comment (markgravity) [inline lib/main.dart:65]: What do you think about `_buildUsers`? 🤔

**Comment by manh-t** [comment]
@doannimble our template won't work like this, instead we will create a project under `examples` folder. Then the task will be implemented there 🙇  I have created a chore issue #44 to create the example project, please wait 😁

**Comment by doannimble** [comment]
@manh-t Yes I see, I will refactor/rebase after your work done 😄

**Comment by luongvo** [comment]
@doannimble please mark this PR as Draft 🙏

**Comment by manh-t** [comment]
Hi @doannimble , I will close this PR since we don't need to implement `example` project anymore 🙏

**Pull request closed without merging**